lib/verbalize/version.rb

```diff
@@ -1,3 +1,3 @@
 module Verbalize
-  VERSION = '2.0.1'.freeze
+  VERSION = '2.0.2'.freeze
```

Comment:
1. Version updates should be done immediately prior to a release, it makes it easier to track what is included in the release itself if it is the last commit.
2. This should be a minor version update, not a patch level update as it adds functionality and is not a bugfix.